tracex_project/extraction/logic/orchestrator.py

```diff
@@ -175,3 +176,13 @@ def update_progress(self, view, current_step, module_name):
             view.request.session["progress"] = percentage
             view.request.session["status"] = module_name
             view.request.session.save()
+
+    def simulate_extraction(self, view):
+        """Simulate the progress of the extraction process."""
+        for current_step, current_module in enumerate(
+            self.configuration.modules, start=1
+        ):
+            self.update_progress(
+                view, current_step, self.configuration.modules[current_module]().name
+            )
+            time.sleep(1)
```

Comment:
I don't know how the presentation is planned by @tkv29, but the appearences of the single steps are really short. Maybe we could increase the time.sleep, but only if Kien wants it, too,

Reply:
True, it could also be 2 or 3 seconds per module, but thats just a matter of changing that 1 to a 2 and could even be done locally to fit Kiens presentation style.

Reply:
Adjusted to two seconds.